packages/e2e-tests/src/specs/editor/fontCheck/fontCheck.js

```diff
@@ -44,6 +44,8 @@ async function createStoryWithTitle(title) {
 }
 
 async function updateFont(fontFamily) {
+  // Open style pane
+  await expect(page).toClick('li[role="tab"]', { text: /^Style$/ });
   await expect(page).toClick('button[aria-label="Font family"]');
   await page.keyboard.type(fontFamily);
   await page.keyboard.press('ArrowDown');
```

Comment:
Hm, GitHub is acting out for me and the last commit I made is just not appearing.

The change for fixing the remaining e2e tests should be adding these lines as the last action inside `replaceFontWithFontPicker` in the `fontCheck.js` e2e tests file.

```
  // Open style pane
  await expect(page).toClick('li[role="tab"]', { text: /^Style$/ });
```

If GItHub starts working again and the commit hasn't appeared, perhaps someone could just add it (as I'll be finishing the day shortly)?

Reply:
it seems to have let me push up! thanks for solving these.